scripts/ci-push.sh

```diff
@@ -6,7 +6,9 @@ source ./scripts/ci-login.sh
 
 ./scripts/build-site.sh
 ./scripts/sync-and-test-bucket.sh
+
 node ./scripts/await-in-progress.js
-./scripts/run-pulumi.sh update
 
+./scripts/run-pulumi.sh update
+./scripts/update-search-index.sh production
```

Comment:
can we add a comment here about the production argument passed in at the end and that it relates to the index/environment being updated in algolia itself?

Reply:
Added a comment to `update-search-index` to clarify what these values are and how they're used.